pkg/actor/deploy.go

```diff
@@ -123,27 +123,23 @@ func (d deploy) Act(ctx context.Context, cluster *resource.Cluster) error {
 		return nil
 	}
 
-	// if we only have one Node we cannot have a PDB
-	// TODO we need to validate this in the CRD API
-	if cluster.Spec().Nodes > 1 {
```

Comment:
Admission controller guarantees that this count >=2

Reply:
Should we have this in a different PR?